package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "jquery": "^3.2.1",
-    "wikibase-data-model": "^4.0.0",
+    "wikibase-data-model": "https://github.com/wmde/WikibaseDataModelJavaScript.git#41c83e27f5e922f9f17ebcc030c6b699b6f6ccc4",
```

Comment:
better to wait until we merge and release there? or we need to remember to chabge it back when we release.

Reply:
Yes, we are waiting for the other one to be merged first and then continue with this. Just checking if tests run. This PR and the ones are in "draft" state.

Reply:
oh I must have missed to see thebdraft state